manifests/0000_26_capi-operator_00_namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Out of interest why are we running this at CVO level 26? I suspect we probably want to run this operator at the same level as MAPI (30) right?

Reply:
mostly because I started from https://github.com/openshift/cluster-cloud-controller-manager-operator/tree/master/manifests and don't really know what these numbers mean - obviously an ordering thing, but
what should be at each step I don't know... I'll change to 30..